cli/cmd/encore/daemon/schema.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS app (
+    root TEXT PRIMARY KEY,
+    local_id TEXT NOT NULL,
+    platform_id TEXT NULL, -- NULL if not linked
+    updated_at TEXT NOT NULL
+);
```

Comment:
Can we please also add a schema version table, just so when we do need to make a breaking change to the schema, it's easier for us to write the migration code

Reply:
Good idea